backend/src/packages/users/user.repository.ts

```diff
@@ -107,7 +107,10 @@ class UserRepository implements Repository {
   }
 
   public delete(): ReturnType<Repository['delete']> {
```

Comment:
we will resolve with boolean, remove all changes in other repositories

Reply:
done